src/TimeService.Data/Filters/FindLeaveTimesFilter.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.TimeService.Data.Filters
+{
+    public class FindLeaveTimesFilter
+    {
+        [FromQuery(Name = "userId")]
+        public Guid? UserId { get; set; }
+
+        [FromQuery(Name = "starttime")]
```

Comment:
strange name, why not startTime or smth else?

Reply:
Because we write everything in small letters